google/api_core/bidi.py

```diff
@@ -276,7 +276,11 @@ def open(self):
         request_generator = _RequestQueueGenerator(
             self._request_queue, initial_request=self._initial_request
         )
-        call = self._start_rpc(iter(request_generator), metadata=self._rpc_metadata)
+        try:
+            call = self._start_rpc(iter(request_generator), metadata=self._rpc_metadata)
+        except exceptions.GoogleAPICallError as exc:
+            self._on_call_done(exc)
+            raise
```

Comment:
What does it raise?

Reply:
It will raise the original exception. 

From https://docs.python.org/3/reference/simple_stmts.html#the-raise-statement,
> If no expressions are present, [raise](https://docs.python.org/3/reference/simple_stmts.html#raise) re-raises the exception that is currently being handled